crates/oxc_wasm/Cargo.toml

```diff
@@ -0,0 +1,48 @@
+[package]
+authors.workspace = true
+description.workspace = true
+edition.workspace = true
+homepage.workspace = true
+keywords.workspace = true
+license.workspace = true
+name = "oxc_wasm"
+repository.workspace = true
+version.workspace = true
+
+[lib]
+crate-type = ["cdylib", "rlib"]
+
+[features]
+default = ["console_error_panic_hook"]
+
+[dependencies]
+miette = {workspace = true, features = ["fancy-no-backtrace"]}
+oxc_allocator = {path = "../oxc_allocator"}
+oxc_ast = {path = "../oxc_ast"}
+oxc_diagnostics = {path = "../oxc_diagnostics"}
+oxc_linter = {path = "../oxc_linter"}
+oxc_parser = {path = "../oxc_parser"}
+oxc_semantic = {path = "../oxc_semantic"}
+serde = {workspace = true, features = ["derive"]}
+serde_json = {workspace = true}
+wasm-bindgen = {version = "0.2", features = ["serde-serialize"]}
```

Comment:
Can you order these (like how the other crates order them), so it's easier to reay?